mentees/README.md

```diff
@@ -0,0 +1,264 @@
+
+There are however a few proven areas you can check out to get the ball rolling:
+
+* If you're completely new to Open Source, there are countless blogs, articles, videos and resources that can be located with a simple web search. There are also popular short courses available, such as those offered by [Linux](https://trainingportal.linuxfoundation.org/learn/course/a-beginners-guide-to-open-source-software-development-lfc102/course-introduction/course-information) or [EDX](https://www.edx.org/course/open-source-software-development-linux-for-developers?index=product&queryID=54e7a9a4b82d487dd017ff72baabd439&position=2) that will help you to learn the fundamentals quickly.
+* Understand that contributions can be either code-based (technical skills required) and non-code (improving documentation, offering feedback), depending on the needs of specific projects.
+* **If you have some basic experience**, check out the following for different opportunities:
+    * Many sites related to CNCF and its projects, like the [CNCF Contributor page](https://contribute.cncf.io/contributors/), will have Contributor guides which outline tasks; as well as lists with several actions you can take to begin your contributor journey.
+    * [CLOTributor](https://clotributor.dev/) is a searchable database with dozens of tasks from [Cloud Native projects](https://www.cncf.io/projects/) you can tackle to build your experience.
+    This can be a great way to familiarise yourself with various projects before choosing to commit to one offered in a specific mentorship program.
+    * Look for issues labeled 'good first issue'. You may also want to start with some of the documentation projects (like the website itself). This is often a good place to start when learning about a new project and looking to contribute.
+
+**Final piece of advice:** The best way to start is contribute immediately; make mistakes and learn from them. This will help you to get familiar with different processes and community members as your work is reviewed, evaluated or approved. 
+
+---
+
+
+### **What is 'Contribution'?**
+Simply put, contribution is about adding value by giving back. The open source community thrives on the efforts of its millions of members globally to help shape, develop and maintain its systems and infrastructure, one incremental step at a time.
+
+Whether you're fixing a minor typo in documentation, or your code is helping to create the next game-changing project, every (approved!) action you make towards development is a critical piece of a bigger puzzle.
+
+Being active in GitHub, meetings, events and blogs or articles are also examples of contribution that show your commitment and consistency, and overall will help both your own journey and the wider community over time.
+
+The best way to start contributing is:
+* Read the [CONTRIBUTING](https://github.com/cncf/tag-contributor-strategy/blob/main/CONTRIBUTING.md) guide.
```

Comment:
I suggest adding, "Learn how to use the project, read the docs, try the tutorial/quickstart, and find ways to gain experience using the project and understanding how it works".